examples/sampling/graphbolt/rgcn/hetero_rgcn.py

```diff
@@ -657,7 +658,7 @@ def main(args):
     )
     parser.add_argument("--num_epochs", type=int, default=3)
     parser.add_argument("--num_workers", type=int, default=0)
-    parser.add_argument("--num_gpus", type=int, default=0)
+    parser.add_argument("--num_gpus", type=int, default=1)
```

Comment:
default to 0?

Reply:
Default to 1 because if there is no GPU available, it will be set back to 0 automatically.

Reply:
Why do we need this parameter? We could simply remove it?